src/components/Person/Person.jsx

```diff
@@ -1 +1,39 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const personPartner = (
+    sex === 'm'
+      ? 'wife'
+      : 'husband'
+  );
```

Comment:
Checklist p2

Reply:
decided to completely remove the extra variable